src/wallet/wallet2.cpp

```diff
@@ -105,7 +105,7 @@ namespace
 struct options {
   const command_line::arg_descriptor<std::string> daemon_address = {"daemon-address", tools::wallet2::tr("Use daemon instance at <host>:<port>"), ""};
   const command_line::arg_descriptor<std::string> daemon_host = {"daemon-host", tools::wallet2::tr("Use daemon instance at host <arg> instead of localhost"), ""};
-  const command_line::arg_descriptor<std::string> password = {"password", tools::wallet2::tr("Wallet password"), "", true};
+  const command_line::arg_descriptor<std::string> password = {"password", tools::wallet2::tr("Wallet password (escape/quote as needed)"), "", true};
```

Comment:
I put that in because someone complained their shell interpreted a $ character, and somehow thought this was monero's fault. Monero's --help page is not really the place for a primer on how to use your OS, and command line interpretation will change depending on your OS/shell, so we don't want to start going there.

Besides, "put the password in quotes" only works in some cases, so it'd be bad advice.